src/bin/edit/draw_filepicker.rs

```diff
@@ -221,6 +221,8 @@ fn draw_file_picker_update_path(state: &mut State) -> Option<PathBuf> {
     };
     if dir != state.file_picker_pending_dir.as_path() {
         state.file_picker_pending_dir = DisplayablePathBuf::from_path(dir.to_path_buf());
+        state.file_picker_pending_dir_revision =
```

Comment:
ah, did we forget to bump the revision when we navigated to `..`?

Reply:
Yep, exactly! I accidentally put the bump in the wrong function.